File: verify/except/any_catch.py
def verify(*args):
    i = int(args[1])
    if i == 0:
        return (0, "Zero\nExit.\n", "")
    elif i == 1 or i == 2:
        return (0, "Something else\nExit.\n", "")
    else:
        return (0, "Exit.\n", "")

data = [
    ["0"],
    ["1"],
    ["2"],
    ["3"],
]

File: assets/except/any_catch.c
int main(int argc, char** argv) {
    int i;
    i = atoi(argv[1]);
    try {
        switch (i) {
            case 0: throw Zero(NULL);
            case 1: throw One(NULL);
            case 2: throw Two(NULL);
        }
    } catch (Zero _) {
        printf("Zero\n");
    } catch (_ _) {
        printf("Something else\n");
    } finally {
        printf("Exit.\n");
    }
}
